originals/s.yaml

```diff
@@ -145,6 +145,20 @@
     - Puzzle
     - Arcade
 
+- name: 'Serious Sam: the Second Encounter'
+  external:
+    wikipedia: 'Serious Sam: The Second Encounter'
+  platform:
+  - DOS
+  - Nintendo Switch
+  - Playstation 4
```

Comment:
```suggestion
  - PlayStation 4
```

Reply:
please use the add game forms, they will help you avoid schema validation errors like this one